.travis.yml

```diff
@@ -6,9 +6,17 @@ node_js:
     - "7"
     - "8"
 sudo: false
-script: "npm test && npm run docs"
+branches:
+    only:
+    - master
+
+# Run npm test always
+# Run npm docs only when its the master branch build and node 8
+script:
+    - "npm test"
+    - 'if [[ "$TRAVIS_PULL_REQUEST" = "false" && "$TRAVIS_BRANCH" = "master" && "$node_js" = "8" ]]; then npm run docs; fi'
```

Comment:
Are you guys suggesting to actually remove the generation of docs from all the build?